crates/ruma-federation-api/src/membership/create_leave_event/v1.rs

```diff
@@ -114,4 +114,12 @@ impl Response {
 
 /// An empty object.
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
 pub struct Empty {}
```

Comment:
Since the spec explicitly says it's an empty object and this endpoint already has a v2, I doubt there will ever be more fields (that also wouldn't fit with the name 😄). I think this one should be exhaustive (though it doesn't hurt to have a constructor).